ci/snaps/snaps_pxe_tmplt.yaml

```diff
@@ -482,6 +485,9 @@ ansible:
       snaps_boot_dir:
         type: string
         value: {{ src_copy_dir }}/snaps-boot
+      check_file:
```

Comment:
This only needs to be checked once per server, not on every interface on each server.

Reply:
@RandyLevensalor good point. Fixed in the latest patch.